web/templates/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="height=device-height, initial-scale=1.0">
+
+    <!-- OpenGraph tags for sharing preview -->
+    <meta property="og:url"
+          content="http://spacypl.sigmoidal.io" />
+    <meta property="og:type"
+          content="website" />
+    <meta property="og:title"
+          content="spaCy.pl" />
+    <meta property="og:description"
+          content="Developing tools for complete Polish language support in spaCy." />
+    <meta property="og:image"
+          content="{{ url_for('static', filename='png/hero.jpg') }}" />
+
+    <title>
+        spaCy.pl
```

Comment:
change to spaCy-pl (to not confuse people)

Reply:
We'll leave that to merging with the other demo (the website) as this is not even visible in this version.